portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,211 @@
+        organization: 'Google',
+        start: 'May 2020',
+        title: 'Step Intern',
+        location: 'Remote'
+      },
+      {
+        organization: 'Intel Corporation',
+        start: 'February 2020',
+        end: 'May 2020',
+        title: 'Software Engineering Intern',
+        location: 'Hillsboro, Oregon',
+        description: [
+          'Researched and documented an initiative to streamline and automate the computer system design process',
+          'Participated in technical meetings with key stakeholders to determine strategic architecture implementation details',
+          'Developed an internal TypeScript-based library to serialize objects with important design information into a new format',
+          'Prototype a hybrid web application to be distributed to different teams and computer vendors, showcasing the functionality of the serialization library'
+        ]
+      },
+      {
+        organization: 'USF Judy Genshaft Honors College IT Department',
+        start: 'January 2019',
+        end: 'January 2020',
+        title: 'Full-stack Web Developer',
+        subtitle: 'ASP.NET C#, MSSQL, Entity Framework, Bootstrap, JavaScript',
+        location: 'Tampa, FL, USA',
+        description: [
+          'Automate academic and administrative processes in the College by working with faculty to develop solutions in an internal web application. Some examples are student form submission and processing, student advising, and graduation ceremony organization',
+          'Use Google Dialogflow to assist in the creation of a virtual assistant for the College to process common advising requests by students',
+          'Lead team in initial phase of codebase modernization, using React.JS, and redesign database structure'
+        ]
+      }
+    ],
+    'Activities': [
+      {
+        organization: 'Society of Hispanic Professional Engineers USF Chapter',
+        start: 'May 2020',
+        title: 'Vice-president of External Affairs',
+        location: 'Tampa, FL, USA'
+      },
+      {
+        organization: 'FIRST Global',
+        start: 'June 2019',
+        title: 'Team Venezuela Mentor',
+        location: 'Remote',
+        description: [
+          'Lead team marketing and external communications with the FIRST Global organization, sponsors, and the community, through the handling of emails, creation and frequent publication content on social media, official team video editing, uniform design, schedule organization, record keeping, etc.'
+        ]
+      },
+      {
+        organization: 'Carrollwood Day School',
+        start: 'August 2019',
+        end: 'January 2020',
+        title: 'FTC Team Nova #11343 Mentor',
+        location: 'Tampa, FL, USA',
+        description: [
+          'Assist in the academic preparation of a team of 5-7 students, as well as the building and programming of robot for season competitions'
+        ]
+      },
+    ]
+  }
+}
+
+function loadContactData(contactData) {
```

Comment:
Per the [styleguide](https://google.github.io/styleguide/jsguide.html#jsdoc), please add a JSDoc at the top of the function (description, parameter, type, return types) to each function.

Reply:
Working on it!

Reply:
Done in latest commit